wiki/BBCode/es.md

```diff
@@ -4,411 +4,343 @@ outdated_translation: true
-
-- Botón dedicado: ![Botón de imagen](img/image.png)
-- Notas:
-  - Si tiene muchas imágenes o imágenes grandes, se recomienda colocarlas dentro de una [Caja](#caja).
-- Sintaxis:
-  - `LINK` es un enlace directo a una imagen.
-  ```
-  [img]LINK[/img]
-  ```
+### Correo electrónico
+
+```
+[email=DIRECCIÓN]texto[/email]
+```
+
+La etiqueta `[email]` crea un hipervínculo en el que se puede hacer clic, que abre un nuevo correo electrónico en el programa de correo predeterminado con el campo de dirección rellenado previamente.
+
+Para crear un enlace, se deben especificar dos argumentos: el argumento `DIRECCIÓN` debe ser una dirección de correo electrónico válida, mientras que `texto` es el texto que se muestra y sirve como hipervínculo. Si el argumento `texto` no está definido, el hipervínculo no se creará correctamente.
+
+### Imágenes
+
+```
+[img]DIRECCIÓN[/img]
+```
+
+La etiqueta `[img]` se usa para incluir imágenes en línea en las publicaciones de foro de osu!. Para usar la etiqueta, los usuarios deben pegar la dirección de la imagen directa (representada por el argumento `DIRECCIÓN` anterior) procedente de un sitio web. Las rutas de archivos locales (por ejemplo, `C:\Users\Name\Pictures\image.jpg`) **no funcionarán**.
+
+*Aviso: la URL de un sitio web **no** es lo mismo que la dirección de una imagen.*
+
+Para obtener una dirección de imagen, uno debe navegar al sitio web en el que se encuentra, pasar el ratón sobre la imagen, hacer clic con el botón derecho en la imagen y seleccionar `Copiar dirección de la imagen`. Luego, la dirección debe copiarse y pegarse entre las etiquetas.
+
+Aunque las imágenes se pueden obtener desde cualquier lugar, osu! recomienda que los usuarios carguen imágenes en sitios de intercambio de imágenes acreditados como [Imgur](https://imgur.com), ya que algunos sitios web no aprecian los enlaces directos a sus imágenes (también conocidos como «hotlinks»).
+
+Botón de la barra de herramientas: ![Botón de imagen](img/image.png "Imagen")
+
+### Mapa de imágenes
+
+```
+[imagemap]
+URL_DE_LA_IMAGEN
+X Y ANCHO ALTURA REDIRECCIÓN TÍTULO
+[/imagemap]
+```
+
+La etiqueta `[imagemap]` se utiliza para integrar uno o más hipervínculos en una imagen en áreas rectangulares.
+
+La imagen, que está incrustada en el sitio web, está representada por el argumento `URL_DE_LA_IMAGEN`. Debe referirse directamente a una imagen alojada en un sitio web.
+
+Para agregar un área en la que se puede hacer clic, se debe insertar una nueva línea que contenga la posición x e y del área, el ancho y el alto del área, así como un enlace para redirigir después del argumento `URL_DE_LA_IMAGEN`. Además, se mostrará un argumento `TÍTULO` opcional al pasar el cursor por el área si se especifica. Un enlace se puede especificar con el argumento `REDIRECCIÓN` u omitirse con `#`. Cada unidad de tamaño (`X`, `Y`, `ANCHO` y `ALTURA`) es un porcentaje (0–100) sin el signo de porcentaje.
```

Comment:
`Toolbar button: ![Imagemap button](img/imagemap.png "Imagemap")` line is missing

Reply:
Botón de la barra de herramientas: ![Botón de mapa de imagen](img/bold.png "Mapa de imagen")